### PR TITLE
September Adélie merge: Networking packages

### PR DESCRIPTION
**Pull request by awilfox**
Various fixes to network-related packages: add tests where available, support IPv6, and two package version bumps: main/apr and main/apr-util.

### CODE REVIEW

**Comment by awilfox** [comment]
Removed the bind commit because it was causing Travis to fail due to large log size.  All it did was add options="!check" anyway, so there is nothing much lost.

**Comment by fabled** [comment]
Cherry-picked some of these. apr build failed, probably concurrency issues (and it's a major update, so needs rebuilding dependencies?).
One mariadb test failed with:
```
      Start 21: mf_iocache
21/59 Test #21: mf_iocache .......................***Failed    0.00 sec
```

**Comment by awilfox** [comment]
apr builds fine with -j8 on my Xeon and builds fine with -j4 on my PowerPC.  I didn't really think about bumping deps for it, really sorry... I know subversion and apache will need bumping.  I can find the rest and make a list tomorrow, and make a PR that bumps them all, if that helps.

Re mariadb: That's an odd test failure.  What architecture was it?

**Comment by fabled** [comment]
Yes, please create separate APR update + dependency rebuild PR. The concurrent build failure happened on ou 24-core builder.

Mariadb test failure is the same 24-core x86_64 build box.

**Comment by ncopa** [comment]
Can you please rebase this PR so we see whats left?
thanks!

File: main/apr-util/APKBUILD [hunk 1]
```diff
@@ -30,16 +30,21 @@ build() {
...
-		--without-gdbm \
-		|| return 1
-	make || return 1
+		--without-gdbm
+	make
+}
+
+check() {
+	cd "$builddir"
+	# testxlate fails because UTF-7 is unsupported
```
Comment (jirutka) [inline main/apr-util/APKBUILD]: This is not very good. Can you somehow easily disable/remove just the failing tests?

**Comment by awilfox** [comment]
So the MariaDB test failure is actually because Adélie uses OpenSSL and Alpine uses LibreSSL.  LibreSSL is not supported in MariaDB [until 10.2.6 or 10.3](https://github.com/MariaDB/server/commit/ccca4f43c92916c347210a7f9a8126f2aa3f6c31).

Any one of the following options must be taken:
* The patch needs to be backported to 10.1
* The port needs to be upgraded to 10.2.6+ or 10.3.0+
* The port needs to use OpenSSL instead of LibreSSL.

**Comment by ncopa** [comment]
fixed with 5fdfcb1f1d2bacb0300539d319d35d3cb0032033, 6556a505d52c020d0bf835a0f1e12afceb303674 and 9f6c5c776a20c9ff350aa839795d1c2bf65fb0ac

**Pull request closed without merging**